packages/ppx_relit/ppx_relit.0.1/opam

```diff
@@ -0,0 +1,23 @@
+opam-version: "1.2"
+name: "ppx_relit"
+version: "0.1"
+maintainer: "Charles Chamberlain <charlespipin@gmail.com>"
+authors: [
+  "Charles Chamberlain <charlespipin@gmail.com"
+  "Cyrus Omar <cyrus.omar@gmail.com"
+]
+homepage: "https://github.com/cyrus-/relit"
+bug-reports: "https://github.com/cyrus-/relit/issues"
+license: "MIT"
+dev-repo: "https://github.com/cyrus-/relit"
+build: [
+  ["jbuilder" "build"]
```

Comment:
Yous should add `”-p”  name` To make sure it is building only the package itself. Also the install and remove will be automatically generated by jbuilder, so you should delete them below

Reply:
This will allow you to also drop the dependency on ocamlfind that seems to be used only to remove the package